redirect-turbo-fame.md

```diff
@@ -0,0 +1,123 @@
+# Live Components + Turbo Streams: Navigating a Frame
+
+Recently, we received a fascinating [question](https://symfonycasts.com/screencast/last-stack/live-components#comment-31545),
+summarized as:
+
+> Suppose I submit a Live component to a [LiveAction](https://symfony.com/bundles/ux-live-component/current/index.html#actions)
+> and redirect from there. How can I make this navigate a `<turbo-frame>` instead
+> of the full page?
+
+Let's learn a bit about how Live Components handles redirects, then see how we can
+use Turbo Streams to navigate a frame instead of the full page. This is team work
+at the max!
+
+## How Live Components Handles Redirects
+
+To answer this, let's create an imaginary Live component that sends a message:
+
+```php
+class SendMessage extends AbstractController
+{
+    #[LiveProp(writable: true)]
+    public string $message = '';
+
+    #[LiveAction]
+    public function send()
+    {
+        // ... send the message
+
+        return $this->redirectToRoute('my-route');
+    }
+}
+```
+
+The template for this component might look like:
+
+```twig
+<div {{ attributes }}>
+    <input data-model="message" />
+
+    <button data-action="live#action data-action-name="send">Send</button>
+```
+
+***TIP
+In an upcoming version of LiveComponents, `data-action-name` will become
```

Comment:
Any specific version known? It would be great to mention the specific version I think